src/domain/products/edit/sections/general/index.tsx

```diff
@@ -99,22 +99,39 @@ type DetailProps = {
 }
 
 const Detail = ({ title, value }: DetailProps) => {
+  const DetailValue = () => {
+    if (Array.isArray(value)) {
+      if (value.length) {
+        return <DelimitedList list={value} delimit={2} />
+      } else {
+        return <p>–</p>
+      }
+    } else {
+      return <p>{value ? value : "–"}</p>
+    }
+  }
+
   return (
     <div className="flex items-center justify-between inter-base-regular text-grey-50">
       <p>{title}</p>
-      <p>{value ? value : "–"}</p>
+      <DetailValue />
     </div>
   )
 }
 
 const ProductDetails = ({ product }: Props) => {
+  const { isFeatureEnabled } = useFeatureFlag()
+
   return (
     <div className="flex flex-col mt-8 gap-y-3">
       <h2 className="inter-base-semibold">Details</h2>
       <Detail title="Subtitle" value={product.subtitle} />
       <Detail title="Handle" value={product.handle} />
       <Detail title="Type" value={product.type?.value} />
       <Detail title="Collection" value={product.collection?.title} />
+      {isFeatureEnabled("product_categories") && (
```

Comment:
suggestion: should that `"product_categories"` be exported from elsewhere as a const and use here?